normandy/base/tests/api/test_api.py

```diff
@@ -235,3 +235,35 @@ def test_logged_out(self, settings):
             "logout_url": settings.OIDC_LOGOUT_URL,
             "github_url": settings.GITHUB_URL,
         }
+
+
+@pytest.mark.django_db
+class TestUserAPI(object):
+    def test_it_works(self, api_client):
+        user = User.objects.first()
+        res = api_client.get("/api/v3/user/")
+        assert res.status_code == 200
+        assert res.data == {
+            "count": 1,
+            "next": None,
+            "previous": None,
+            "results": [
+                {
```

Comment:
@mythmon do we care about more than this info when creating users?

Reply:
I don't care about any other information than this, though we should make sure that the username value is the same as the email. We don't need to expose it though.

Reply:
We'll also need group information somewhere. Maybe here? I'm not sure if we want to publish that though, so we might need to use a clever field that only shows if you have permissions.